python/dgl/nn/pytorch/conv/graphconv.py

```diff
@@ -7,6 +7,136 @@
 from .... import function as fn
 from ....base import DGLError
 from ....utils import expand_as_pair
+from ....transform import reverse
+from ....convert import block_to_graph
+from ....heterograph import DGLBlock
+
+class EdgeWeightNorm(nn.Module):
```

Comment:
Do you want to expose this module to users? If so, you need to import it in `python/dgl/nn/pytorch/conv/__init__.py ` and update `docs/source/api/python/nn.pytorch.rst`.

Reply:
Good catch, I forgot about the rst.